libs/experimental/pyproject.toml

```diff
@@ -33,6 +33,8 @@ setuptools = "^67.6.1"
 # dependencies used for running tests (e.g., pytest, freezegun, response).
 # Any dependencies that do not meet that criteria will be removed.
 pytest = "^7.3.0"
+duckdb = "^0.8.1"
```

Comment:
can we make these optional? don't think we want these as required dependencies

Reply:
Just removed them.